controllers/controllers.js

```diff
@@ -1,6 +1,6 @@
 const { response } = require('../app');
 const app = require('../app')
-const {selectTopics, selectArticleByID, selectUsers, updateVote} = require('../models/models')
+const {selectTopics, selectArticleByID, selectUsers, updateVote, selectArticles} = require('../models/models')
```

Comment:
Again, same as controller I would separate out the different data sets to their own models e.g topicsModel, articlesModel